test/annexB/language/statements/for-in/labeled-fn-stmt-lhs.js

```diff
@@ -0,0 +1,11 @@
+// Copyright (C) 2016 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+description: It is a Syntax Error if IsLabelledFunction(Statement) is true.
+negative: SyntaxError
+id: sec-labelled-function-declarations
+es6id: B.3.2
+---*/
+
+var x;
```

Comment:
You are correct. Three other tests for SyntaxErrors also needlessly included variable declarations. I've removed them all in 9be1481